### PR TITLE
Fix install recipe

### LINKED ISSUE #13: README Please provide alternate recipe for adding repos
curl https://install.citusdata.com/community/deb.sh | sudo bash

Encouraging people to blindly execute content from the internet as root is as dangerous as it is convenient - on both security grounds and failure modes. Please provide alternate default instructions to add your repos.

(I hasten to add that this project looks awesome and I’ll be looking to see if we can drop some homegrown equivalents - thank you for opening this!)

### PR DESCRIPTION
**Pull request by DimCitus**
Add the manual steps to install our repositories and packages, in addition to the quick install method requiring the scary `curl | sudo bash` stanza.

Fixes #13.

### CODE REVIEW

File: docs/install.rst [hunk 1]
```diff
@@ -12,6 +12,15 @@ PostgreSQL 11.
 Ubuntu or Debian
 ----------------
 
+Quick install
+~~~~~~~~~~~~~
+
+The following installation method downloads a bash script that automates
+several steps. The full script is available for review at our `package cloud
+installation instructions page`__ url.
```
Comment (mtuncer) [inline docs/install.rst]: may drop `url` word

Comment (mtuncer) [reply docs/install.rst]: more like
package cloud installation instructions __ page.

File: docs/install.rst [hunk 2]
```diff
@@ -50,15 +116,15 @@ Here's a sample output from the command:
 
    $ pg_autoctl show systemd
```
Comment (mtuncer) [inline docs/install.rst]: `pg_autoctl show systemd` errors out when `--pgdata` is not provided. We are not setting `--pgdata` in the tutorial. Perhaps we should either set environment variable or provide the value

File: docs/install.rst (hunk 1 above)
Comment (DimCitus) [reply docs/install.rst]: Fixed that way, thanks!

File: docs/install.rst (hunk 2 above)
Comment (DimCitus) [reply docs/install.rst]: Fixed!

**Pull request merged**